spec/mailers/claim_mailer_spec.rb

```diff
@@ -16,4 +16,20 @@
       expect(mail.body.encoded).to match("Your unique reference is #{claim.reference}. You will need this if you contact us about your claim.")
     end
   end
+
+  describe "#approved" do
+    let(:claim) { create(:claim, :submitted, first_name: "John", middle_name: "Fitzgerald", surname: "Kennedy") }
+    let(:mail) { ClaimMailer.approved(claim) }
+
+    it "renders the headers" do
+      expect(mail.subject).to eq("Your claim to get your student loan repayments back has been approved")
```

Comment:
One tiny suggestion, I think it might be worthwhile including their reference number again in the subject e.g
`Your claim to get back your student loan repayments back has been approved, reference number: {REF NUMBER}